pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -47,6 +62,25 @@ let
     makeFlags = [ "PREFIX=$(out)" ]
       ++ lib.optional rustSupport "PURE=--rust";
 
+    doCheck = stdenv.isLinux;
```

Comment:
Please leave a comment as to why you're disabling tests on not-Linux.

Reply:
Added and amended!